Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:lts
+
+ENV NODE_ENV="docker"
+
+WORKDIR /dolos
+
+COPY package.json ./
+
+RUN npm install --quiet
+
+RUN npm install nodemon -g
+
+COPY . .
+
+EXPOSE 3000
+
+CMD nodemon -L --watch . src/app.js
```

Comment:
why running a development command in docker container ? we won't be hot reloading right ?

Reply:
When we create a dockerfile, the docker build the same app but with completely different location to keep all services together and we need to be able to sync this location with src local files. 
There is not difference between this command or npm run dev, but I keep more simple for other people learning this.